packages/block-library/src/cover/style.scss

```diff
@@ -63,6 +63,11 @@
 		@supports (-webkit-overflow-scrolling: touch) {
 			background-attachment: scroll;
 		}
+
+		// Remove the appearance of scrolling based on OS-level animation preferences.
+		@media (prefers-reduced-motion: reduce) {
+			background-attachment: local;
```

Comment:
Thanks @ryelle! This is a great, simple change. One tiny question: Is there a reason you chose `local` here instead of the default `scroll`? It looks like we're falling back to `scroll` above, so it might make sense to use that hear too.

Reply:
I think I just misunderstood the docs for which was the default value– you're right, it should be scroll - I'll update that 👍

Reply:
Updated the value.